src/fs.zig

```diff
@@ -1608,8 +1608,13 @@ pub const PathName = struct {
 
         // Strip off the extension
         if (strings.lastIndexOfChar(base, '.')) |dot| {
-            ext = base[dot..];
-            base = base[0..dot];
+            if (strings.endsWith(base, "bun.lock")) {
```

Comment:
should we make sure it's the entire file name? like:
```suggestion
            if (strings.endsWith(base, std.fs.path.sep_str ++ "bun.lock")) {
```

Reply:
or actually why are we settings extension to `"bun.lock"`?

Reply:
btw i found out `base` was just filename, so was easy to ensure entire file name

Reply:
> or actually why are we setting extension to `"bun.lock"`?

oops i didn't look hard enough into the source, i have found a way safer option now. Thanks!